api/controller/pin.js

```diff
@@ -1,88 +1,80 @@
-import { createPin } from "../model";
+import { createPin, deletePin, getPinById, updatePin } from "../model";
 import User from "../model/user";
 
 const addNewPin = async (req, res) => {
-  const { body = {} } = req;
+  const { body } = req;
+
+  if (!body.latitude || !body.longitude || !body.title) {
+    return res.status(400).json({ message: "Latitude, longitude, and title are required." });
+  }
```

Comment:
Can you do this validation using express-validator middleware ??

Reply:
sure I will try it